package.json

```diff
@@ -30,11 +30,10 @@
 		"test:js": "cross-env BABEL_ENV=test NODE_ENV=test jest --config=tests/js/jest.config.js --passWithNoTests",
 		"test:js:watch": "npm run test:js -- --watch",
 		"test:storybook": "npm run test:js -- --testMatch '<rootDir>/.storybook/*.test.js'",
-		"pretest:e2e": "./bin/local-env/env-check.sh && ./tests/e2e/bin/test-docker-connectivity",
```

Comment:
@aaemnnosttv suggested removing the pretest command as it's not needed to be run from root. Looking into it I can't see it being run anywhere at all so we could remove it completely.